.github/workflows/master.yml

```diff
@@ -556,9 +556,9 @@ jobs:
         run: |
           yarn add prettier@^2.6.2
           yarn checkformat
-  sonarcloud:
+  sonarcloud-and-codecov:
     if: github.repository == 'apache/submarine' && github.event_name == 'push' && github.ref == 'refs/heads/master'
```

Comment:
sorry, I forgot we only trigger sonarcloud on push. we should add a new job for `codecov` to trigger it in pull-request.

Reply:
Ok, I just changed the position of  if statement into sonarcloud's own step. Thank's Kevin